### ISSUE TITLE
UDP Packet Losses

### ISSUE THREAD

**Issue by gretamosa** [repeatedly/fluent-plugin-netflow#25]
I have installed ruby 2.3.1 with fluent 0.17 and this netflow plugin. Then, I generate random netflow flows with mshindo netflow generator and according with the README document.

Once all is started, I monitor udp traffic and I can see udp packet losses every second. Do you have a specific buffer size in your OS or similar?

Regards,

**Comment by repeatedly** [comment]
> Once all is started, I monitor udp traffic and I can see udp packet losses every second. 

Does it mean netflow input plugin receives broken packet?

**Comment by gretamosa** [comment]
Yes, it receives udp packets with errors or broken. I monitor udp traffic with netstat -su (udp section).
